### ISSUE TITLE
CSS problem at the beginning

### ISSUE THREAD

**Issue by ishabaral** [JankariTech/blog#158]
There is a problem with CSS at the start. If the 1st paragraph is short, the text doesn't align properly.

![Screenshot from 2024-09-10 11-44-37](https://github.com/user-attachments/assets/c1b3de2e-5c05-4373-a0ff-1c2d70346885)

**Comment by koebel** [comment]
Good catch, yes let's remove these initials, they really make it hard to read the text. And in some cases as displayed in the screenshot above completely mess up the layout.
